drag_and_drop_v2/templates/html/drag_and_drop_edit.html

```diff
@@ -20,7 +20,7 @@
                     </label>
 
                     <h3>{% trans "Maximum score" %}</h3>
-                    <input class="weight" value="1" value="{{ self.weight }}"/>
+                    <input class="weight" value="1" value="{{ self.weight }}" />
```

Comment:
Not your fault, but while I notice this, can you please add `type="number"` to this input?